client/www/scripts/modules/tracing/tracing.module.js

```diff
@@ -1 +1,5 @@
 var Tracing = angular.module('Tracing', []);
+var TRACING_CONST = {
+  FF_UNTAGGED_WATERFALLS: 'feature:untagged'
```

Comment:
Please fix. Use 'traces:untagged' as the flag as specified in the issue instead of 'feature:untagged'